packages/sqlite3/meta.yaml

```diff
@@ -27,16 +27,48 @@ build:
       "Modules/_sqlite/util.c"
     )
 
-    embuilder build sqlite3 --pic
+    export SQLITE_WASM_VERSION=3450300
+    export SQLITE_WASM_YEAR=2024
+    wget -O sqlite.zip https://www.sqlite.org/${SQLITE_WASM_YEAR}/sqlite-src-${SQLITE_WASM_VERSION}.zip
```

Comment:
Would you mind updating the [sqlite3 version in Emscripten](https://github.com/emscripten-core/emscripten/blob/e6ed9d3fc2821a72cc8ded4a02cac72b9f306f88/tools/ports/sqlite3.py#L8-L10) instead? It would be a two-step process, as Emscripten should first update the sqlite3 version, then Pyodide need to update the emscripten version, but it reduces our maintenance burden.